boilerplate/web_modules/layouts/index.js

```diff
@@ -1,5 +1,12 @@
-export Page from "./Page" // default layout
-export PageError from "./PageError"
+import Page from "./Page" // default layout
+import PageError from "./PageError"
 
 // you can add your own layout here
-// eg: export Post from "Post"
+// eg: import Post from "Post"
+
+const layouts = {
```

Comment:
I don't really like this breaking change (not necessary). Maybe in the dev server we should do something smarter instead of just .default?